### DISCUSSION TITLE
Table names in the demo example not visible in Chrome Tab(version-124.0.6367.79)

### DISCUSSION THREAD

**Discussion by jibinjoy8897** [clientIO/joint#2637] [Q&A]
### Introduction

URL - https://www.jointjs.com/demos/database
Issues :
1) Table names are not visible on the mentioned Chrome version in Incognito mode.
2) Tables names are getting hidden on repeated reloads in normal Chrome tab 

PFA Screenshots
![JOINTJSキャプチャ](https://github.com/clientIO/joint/assets/56633789/1848fcca-42d2-440d-bfcb-74f0524fce68)
![JOINTJSキャプチャ2](https://github.com/clientIO/joint/assets/56633789/6eef7610-2fc7-4413-b9a8-e05a69b2748b)
![SECRETMODE_JOINTJSキャプチャ](https://github.com/clientIO/joint/assets/56633789/1f231f32-18eb-434a-befb-367fa1c587d6)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Can't reproduce on Mac `Version 124.0.6367.119 (Official Build) (arm64)`.

What's your OS?
